enterprise/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/EnterpriseCompilerFactory.scala

```diff
@@ -57,15 +57,15 @@ class EnterpriseCompilerFactory(community: CommunityCompilerFactory,
       val dispatcher =
         if (workers == 1) new SingleThreadedExecutor(morselSize)
         else {
-          val numberOfThreads = if (workers == 0) Runtime.getRuntime.availableProcessors() else workers
+          val numberOfThreads = if (workers == 0) java.lang.Runtime.getRuntime.availableProcessors() else workers
           val jobScheduler = graph.getDependencyResolver.resolveDependency(classOf[JobScheduler])
           val executorService = jobScheduler.workStealingExecutor(JobScheduler.Groups.cypherWorker, numberOfThreads)
 
           new ParallelDispatcher(morselSize, numberOfThreads, executorService)
```

Comment:
Maybe not in the scope of this PR, but wouldn't it make sense for dispatcher to be a lazy val?